libs/base/math.ts

```diff
@@ -1,4 +1,14 @@
-namespace Math {    
+namespace Math {
+
+    export let E = 2.718;
+    export let LN2 = 0.693;
+    export let LN10 = 2.303;
+    export let LOG2E = 1.443;
+    export let LOG10E = 0.434;
+    export let PI = 3.14159;
```

Comment:
We need the full precision used in JavaScript.

Reply:
I updated the values with the values returned from a Chrome console. Is that what you meant?